.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+<!--
+If you are reporting a new issue, make sure that we do not have any duplicates
+already open. You can ensure this by searching the issue list for this
+repository. If there is a duplicate, please close your issue and add a comment
+to the existing issue instead.
+
+If you suspect your issue is a bug, please edit your issue description to
+include the BUG REPORT INFORMATION shown below. If you fail to provide this
+information within 7 days, we cannot debug your issue and will close it. We
+will, however, reopen it if you later provide the information.
+
+---------------------------------------------------
+BUG REPORT INFORMATION
+---------------------------------------------------
+Use the commands below to provide key information from your environment:
+You do NOT have to include this information if this is a FEATURE REQUEST
+-->
+
+**Description**
+
+<!--
+Briefly describe the problem you are having in a few paragraphs.
+Please include name of the linux distribution, its version and the
+version of the kernel via 'uname -a'.
+-->
+
+**Steps to reproduce the issue:**
+1.
+2.
+3.
+
+**Describe the results you received:**
+
+
+**Describe the results you expected:**
+
+
+**Output of `buildah version`:**
+
+```
```

Comment:
Also need the OS/Kernel Version/rpm -q buildah
cat /etc/containers/storage.conf

Reply:
Some of this was requested in the description section.  I've removed it from there and have added the above suggestions in the same manner as buildah version.  Please let me know if it needs further tweaking.